resources/assets/js/model/reducer/post/index.js

```diff
@@ -19,19 +19,23 @@ import result from 'lodash/result';
 import DataMapping from '~/utils/redux/components/modal/dataMapping';
 
 let commentTable, studyTable;
+let initializedTable = false;
 let initializedStats = false;
 
 const postReducer = (state = initState, action) => {
     switch (action.type) {
         case INIT_POST_OPTION_DEPARTMENT: {
-            commentTable = DataMapping.transObjToKeysTable(
-                state.form['comment'],
-                DataMapping.action.GET_INIT_COMMENT
-            );
-            studyTable = DataMapping.transObjToKeysTable(
-                state.form['study'],
-                DataMapping.action.GET_INIT_STUDY
-            );
+            if (!initializedTable) {
```

Comment:
用`initializedComment`比較不會confuse
btw這邊可以直接用這個嗎?
```suggestion
            if (!commentTable) {
```

Reply:
這裡有兩張表哦，study以及comment

Reply:
了解